tests/FSharp.Data.GraphQL.IntegrationTests.Server/Helpers.fs

```diff
@@ -22,32 +18,20 @@ module StringHelpers =
         |> Array.ofSeq
         |> Encoding.UTF8.GetString
 
-    let utf8Bytes (str : string) =
-        str |> Encoding.UTF8.GetBytes
+    let utf8Bytes (str : string) = str |> Encoding.UTF8.GetBytes
 
-    let isNullOrWhiteSpace (str : string) =
-        String.IsNullOrWhiteSpace(str)
+    let isNullOrWhiteSpace (str : string) = String.IsNullOrWhiteSpace (str)
 
-[<AutoOpen>]
-module JsonHelpers =
-    let tryGetJsonProperty (jobj: JObject) prop =
-        match jobj.Property(prop) with
-        | null -> None
-        | p -> Some(p.Value.ToString())
-
-    let getJsonSerializerSettings (converters : JsonConverter seq) =
-        JsonSerializerSettings()
-        |> tee (fun s ->
-            s.Converters <- List<JsonConverter>(converters)
-            s.ContractResolver <- CamelCasePropertyNamesContractResolver())
-
-    let getJsonSerializer (converters : JsonConverter seq) =
-        JsonSerializer()
-        |> tee (fun c ->
-            Seq.iter c.Converters.Add converters
-            c.ContractResolver <- CamelCasePropertyNamesContractResolver())
-
-    let private converters : JsonConverter [] = [| OptionConverter() |]
-
-    let jsonSettings = getJsonSerializerSettings converters
-    let jsonSerializer = getJsonSerializer converters
\ No newline at end of file
+    
+    open Microsoft.Extensions.DependencyInjection
+    open Microsoft.Extensions.Logging
+
+    type IServiceProvider with
+        member sericeProvider.CreateLogger (``type`` : Type) =
+            let loggerFactory = sericeProvider.GetRequiredService<ILoggerFactory>()
+            loggerFactory.CreateLogger(``type``)
+    open Microsoft.FSharp.Quotations.Patterns
```

Comment:
```suggestion

    open Microsoft.FSharp.Quotations.Patterns
```

Reply:
Fixed in 324eb02 (Relocating logging helper code slightly.)